courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb

```diff
@@ -34,36 +34,10 @@
    "metadata": {},
```

Comment:
<p>Leave the original line, "Now, let's confirm the date parsing".</p><p> </p><p>Replace line 1 with, "Here, we are creating a new dataframe called "grouped_data" and grouping by on the column "Make".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
Updated the same.